qmlti/config.php

```diff
@@ -31,7 +31,7 @@
 define('DEBUG_MODE', TRUE);
 define('SECURE_COOKIE_ONLY', FALSE);
 define('ADMINISTRATOR_ROLE', 'LTI_INSTRUCTOR');
-define('WEB_PATH', '/LTI');  // enter the path starting with a "/" but without a trailing "/"; only required if the automated version does not work
+define('WEB_PATH', '');  // enter the path starting with a "/" but without a trailing "/"; only required if the automated version does not work
```

Comment:
This change is deploying back to the original build as defined in /master branch. I added the /LTI modifier needlessly, and I'm removing it as part of this branch merge.